geotiepoints/modisinterpolator.py

```diff
@@ -86,15 +86,18 @@ def get_corners(arr):
 
 class ModisInterpolator():
 
-    def __init__(self, cres, fres):
+    def __init__(self, cres, fres, cscan_full_width=None):
         if cres == 1000:
```

Comment:
Would it be more explanatory to introduce an optional `product_level` argument? The `cscan_full_width` will be then fixed according to its value.

Reply:
Maybe. Although I think @BENR0 reported that it could also be 269 pixels in some cases, so maybe providing the actual number of pixels is to be prefered ?

Reply:
Plus it doesn't require to pass an extra argument to `modis_5km_to_1km` since the size is deduced from longitude argument. Let's keep it hence as you've suggested.

Reply:
Yes I think it is better to use the cscan_full_width instead of the level argument due to what @mraspaud said. Also this can possibly directly be used in the readers then because the scan width, as far as I remember for the mod35_l2 product, is in the attributes.